content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -240,6 +240,24 @@ Now it's time to take a look at the metrics we're working with.
 _If you don't have a number of training epochs defined and you don't terminate
 the process, the experiment will run for 100 epochs._
 
+### Caching checkpoints
+
+We can automatically push the checkpoints' code & data to your Git & DVC remotes
+while an experiment is running. To enable this, two environment variables need
+to be set:
+
+- `DVC_EXP_AUTO_PUSH`: Enable auto push (`true`, `1`, `y`, `yes`)
+- `DVC_EXP_GIT_REMOTE`: Git repository (can be a URL or a name such as `origin`,
+  `myremote`, etc.)
+
+Note that a `dvc remote default` is also needed so that the corresponding data can
+be pushed. With this configuration, `dvc exp push` will be done automatically
+after every iteration.
```

Comment:
```suggestion
Note that a `dvc remote default` is also needed so that the corresponding data
can be pushed. With this configuration, `dvc exp push` will be done
automatically after every iteration.
```